common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/io/impl/AbstractGeoInstanceWriter.java

```diff
@@ -65,6 +67,16 @@ public String getCustomEPSGPrefix() {
 		return getParameter(PARAM_CRS_CODE_FORMAT).as(String.class);
 	}
 
+	@Override
+	public void setWindingOrder(EnumWindingOrderTypes windingOrderType) {
+		setParameter(PARAM_UNIFY_WINDING_ORDER, Value.of(windingOrderType));
+	}
+
+	@Override
+	public EnumWindingOrderTypes getWindingOrder() {
+		return getParameter(PARAM_UNIFY_WINDING_ORDER).as(EnumWindingOrderTypes.class);
```

Comment:
I don't think that will work. I think you have to use the string value (`as(String.class)`) and then use the `valueOf()` method of the enum type.